grafana/common/PG_Details.json

```diff
@@ -2093,55 +2169,46 @@
       },
       "type": "graph",
       "xaxis": {
-        "buckets": null,
         "mode": "time",
-        "name": null,
         "show": true,
         "values": []
       },
       "yaxes": [
         {
           "$$hashKey": "object:2327",
           "format": "short",
-          "label": null,
           "logBase": 1,
-          "max": null,
-          "min": null,
           "show": true
         },
         {
           "$$hashKey": "object:2328",
           "format": "short",
-          "label": null,
           "logBase": 1,
-          "max": null,
-          "min": null,
           "show": true
         }
       ],
       "yaxis": {
-        "align": false,
-        "alignLevel": null
+        "align": false
       }
     }
   ],
   "refresh": "15m",
-  "schemaVersion": 27,
+  "schemaVersion": 37,
   "style": "dark",
   "tags": [],
   "templating": {
     "list": [
       {
-        "allValue": null,
         "current": {
           "selected": false,
-          "text": "crunchy",
-          "value": "crunchy"
+          "text": "charlie",
```

Comment:
when exporting should we be using 'cluster_name: crunchy' ?

Reply:
These values don't really matter too much. It's just whatever happened to be populated in the drop downs at that time. Whenever a user imports them, the first time they load they are reset as soon as the variable is evaluated.